src/app/content/components/Toolbar.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { SFC } from 'react';
+import styled from 'styled-components';
+import {ListOl} from 'styled-icons/boxicons-regular/ListOl';
+import {Search} from 'styled-icons/fa-solid/Search';
+import {Print} from 'styled-icons/fa-solid/Print';
+import theme from '../../theme';
+
+const ListIcon = styled(ListOl)`
+  height: 2rem;
+  width: 2rem;
+  color: #027EB5;
+  margin-right: 0.7rem;
+
+  :hover {
+    color: #0064A0;
+  }
+
+`;
+
+const SearchIcon = styled(Search)`
+  height: 2rem;
+  width: 2rem;
+  color: #818181;
+  margin-right: 0.7rem;
+  position: absolute;
+
+  @media (max-width: 700px) {
+    height: ${theme.iconStyles.mobile.height};
```

Comment:
this height i'm not sure belongs in the theme. would it be used anywhere outside of this file? maybe just declare it as a const at the top of this file?